### ISSUE TITLE
Coverage is not displayed for tests with @QuarkusTest

### ISSUE THREAD

**Issue by mhagnumdw** [microsoft/vscode-java-test#1719]
Hi!

Simple project to demonstrate that the coverage of tests annotated with `@QuarkusTest` (`GreetingResourceTest.java`) is not displayed in vscode, but the coverage of the other tests (`CollectionUtilsTest.java`) is displayed: https://github.com/mhagnumdw/quarkus-vscode-java-test

![image](https://github.com/user-attachments/assets/91178e9d-5427-48c4-9d65-31b58de9c7b7)

edit: On the maven command line the coverage is ok.

**Comment by jacksoncastro** [comment]
I have the same problem here. Can anyone help?

**Comment by jdneo** [comment]
Looks like it's due to the jacoco version unalignment. The project's dependency contains jacoco `0.8.12`. While the jacoco embedded in java test runner extension is `0.8.11`. I tested locally to upgrade the jacoco version to `0.8.12` and it works.

So I need to update the jacoco version and make a pre-release to unblock you.

Another thing that can be considered is to read the existing exec file which is configured by user. Since I noticed that the `QuarkusTest` will always generate jacoco data file under `target` folder.
